tests/test_minkowski_space.py

```diff
@@ -26,65 +23,54 @@ def setUp(self):
         self.metric = self.space.metric
         self.n_samples = 10
 
-    @classmethod
-    def setUpClass(cls):
-        os.environ['GEOMSTATS_BACKEND'] = 'tensorflow'
-        os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
-        importlib.reload(gs)
-
-    @classmethod
-    def tearDownClass(cls):
-        os.environ['GEOMSTATS_BACKEND'] = 'numpy'
-        importlib.reload(gs)
-
     def test_belongs(self):
         point = self.space.random_uniform()
         belongs = self.space.belongs(point)
-        expected = tf.convert_to_tensor([[True]])
+        expected = gs.array([[True]])
 
-        with self.test_session():
+        with self.session():
             self.assertAllClose(gs.eval(belongs), gs.eval(expected))
```

Comment:
If we need the gs.eval here, maybe abstract it in the method of the class geomstats.tests.TestCase?